hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -176,6 +176,10 @@ public void join() throws InterruptedException {
     auditMap.put("owner", owner);
     List<AllocatedBlock> blocks = new ArrayList<>(num);
     boolean auditSuccess = true;
+    LOG.info("Allocating blocks {} of size {}, with excludeList: " +
```

Comment:
Allocating blocks {} -> Allocating {} blocks might be better in terms of readability.